test/signs/realtime_test.exs

```diff
@@ -1305,7 +1309,7 @@ defmodule Signs.RealtimeTest do
       end)
 
       expect(Engine.Predictions.Mock, :for_stop, fn _, _ -> [] end)
-      expect(Engine.ScheduledHeadways.Mock, :display_headways?, 2, fn _, _, _ -> false end)
+      expect(Engine.ScheduledHeadways.Mock, :display_headways?, 0, fn _, _, _ -> false end)
```

Comment:
Wanted to just double check here, for early AM and this new logic, we aren't showing headways anymore right?

Reply:
I don't think this should affect how the early AM feature works with headways unless I missed something in the task description or a discussion? Headways in the early AM only start showing when we're one [lower headway bound] away from the first scheduled departure.